src/lib/badge/_badge-theme.scss

```diff
@@ -137,6 +137,10 @@ $mat-badge-large-size: $mat-badge-default-size + 6;
     white-space: nowrap;
     text-overflow: ellipsis;
     pointer-events: none;
+
+    ._mat-animation-noopable &, .ng-animate-disabled & {
```

Comment:
I've had this one open for a while now that handles the badge: https://github.com/angular/material2/pull/11408.

Reply:
I see, though it does not handle the `@.disabled` binding and is not aligned with the way we go with #12829.